### PR TITLE
More useful features!

### PR DESCRIPTION
**Pull request by kind-serge**
- Overloads that take in `Tuple'2`
- Ability to add/retrieve relevant info to/from `Exception.Data`
- Auto add relevant info to `Exception.Data` on first-chance exception
- Upgrade to .NET Std 2.0

### CODE REVIEW

File: src/Contextually/Relevant.cs [hunk 1]
```diff
@@ -61,6 +67,96 @@ public static IDisposable Info(NameValueCollection info, string containerName =
...
+        /// Starts a new Info block with a set of values.
+        /// </summary>
+        /// <param name="tuple1">A name-value tuple this Info block represents.</param>
+        /// <param name="tuple2">A name-value tuple this Info block represents.</param>
+        /// <returns>An <see cref="IDisposable"/> object for use with a using block.</returns>
+        public static IDisposable Info(
+            (string name, string value) tuple1,
+            (string name, string value) tuple2) =>
+            Info(new NameValueCollection(2)
+            {
```
Comment (aidapsibr) [inline src/Contextually/Relevant.cs:104]: Why bother with this versus params?

File: src/Contextually/Relevant.InfoAutoAddedToExceptions.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿#if !NETSTANDARD1_3
```
Comment (aidapsibr) [inline src/Contextually/Relevant.InfoAutoAddedToExceptions.cs:1]: Let's drop 1.3 support in favor of 2.0 and remove this

File: src/Contextually/Contextually.csproj [hunk 3]
```diff
@@ -17,5 +17,6 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Condition="'$(TargetFramework)'=='netstandard1.3'" Include="System.Collections.Specialized" Version="4.3.0" />
```
Comment (aidapsibr) [inline src/Contextually/Contextually.csproj:19]: Can drop this once we go to 2.0

**Review by aidapsibr — CHANGES_REQUESTED** [review]

File: src/Contextually/Relevant.cs (hunk 1 above)
Comment (kind-serge) [reply src/Contextually/Relevant.cs:104]: The memory allocation. `params` is a syntactic sugar that transpiles into an array.

File: src/Contextually/Relevant.InfoAutoAddedToExceptions.cs (hunk 2 above)
Comment (kind-serge) [reply src/Contextually/Relevant.InfoAutoAddedToExceptions.cs:1]: Why then to support Fx 4.5.2?
P.S. that would require major version update.

**Review by aidapsibr — APPROVED** [review]

**Comment by aidapsibr** [comment]
I just have to get around to generate the packages now...

**Pull request merged**